src/app/shared/components/cluster-summary/template.html

```diff
@@ -468,6 +468,11 @@
             </div>
           </km-property>
 
+          <km-property *ngIf="operatingSystemProfile">
+            <div key>Operating System Manager</div>
```

Comment:
```suggestion
            <div key>Operating System Profile</div>
```

Reply:
changed in the last commit
https://github.com/kubermatic/dashboard/pull/4602/commits/0fece656f1bf38e9726c31e5e484348540fa9884